### ISSUE TITLE
opts is undefined error

### ISSUE THREAD

**Issue by karuppiah7890** [feross/simple-peer#105]
This error is occurring when I use 

```javascript
var peer2 = new SimplePeer();
```

as given in the simpler example code in README. Later I dug into index.js to find a problem in [this line](https://github.com/feross/simple-peer/blob/master/index.js#L20). Due to this, I had to change the above code to 

```javascript
var peer2 = new SimplePeer({});
```

for it to work.

And just now I checked the docs, and looks like `opts` is optional. So, I dug into the code and found [this line](https://github.com/feross/simple-peer/blob/master/index.js#L26) then, I think you need to put this as the first line of the function.

**Comment by feross** [comment]
Oh, thanks. This is a recent bug I just introduced. I'll fix.

**Comment by feross** [comment]
Released as 6.1.0
